### PR TITLE
PMM-6377 Switch to mongodb_exporter v2

### PR DESCRIPTION
**Pull request by Sudokamikaze**

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1058-ce2707e
client docker - perconalab/pmm-client-fb:PR-1058-ce2707e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1058-ce2707e.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1058-ce2707e
client docker - perconalab/pmm-client-fb:PR-1058-ce2707e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1058-ce2707e.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1058-ce2707e
client docker - perconalab/pmm-client-fb:PR-1058-ce2707e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1058-ce2707e.tar.gz

File: build/bin/build-client-binary [hunk 1]
```diff
@@ -24,6 +24,8 @@ gobuild_component() {
     fi
     if [ ${component} = 'pmm-admin' -o ${component} = 'pmm-agent' ]; then
         build_command="cd /tmp/go/src/${extract_path} && make release"
+    elif [ ${component} = 'mongodb_exporter' ]; then
+        build_command="cd /tmp/go/src/${extract_path} && go get -u ./... && make build"
```
Comment (AlekSi) [inline build/bin/build-client-binary:28]: Nope, we can't use `go get` – that will use the latest commit instead of commit in submodule

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: build/bin/build-client-binary (hunk 1 above)
Comment (AlekSi) [reply build/bin/build-client-binary:28]: /cc @EvgeniyPatlan ^

**Comment by AlekSi** [comment]
@alexanderblinkov-percona It is not a draft as it is not a FB – there are changes in build scripts that should be merged

File: build/bin/build-client-binary (hunk 1 above)
Comment (Sudokamikaze) [reply build/bin/build-client-binary]: @AlekSi Thank you, removed in d310358

File: build/bin/build-client-binary [hunk 2]
```diff
@@ -24,6 +24,8 @@ gobuild_component() {
     fi
     if [ ${component} = 'pmm-admin' -o ${component} = 'pmm-agent' ]; then
         build_command="cd /tmp/go/src/${extract_path} && make release"
+    elif [ ${component} = 'mongodb_exporter' ]; then
+        build_command="cd /tmp/go/src/${extract_path} && make build"
```
Comment (AlekSi) [inline build/bin/build-client-binary:28]: @percona-csalguero @EvgeniyPatlan any objections to rename that to `make release` just for consistency?

Comment (AlekSi) [reply build/bin/build-client-binary:28]: Ah, we have release already – for Go Releaser. Please ignore that

**Comment by Sudokamikaze** [comment]
@AlekSi please note, that ce2707e should be reverted before merging, it's for FB purpose

**Comment by AlekSi** [comment]
Yep, doing that right now

File: .gitmodules [hunk 3]
```diff
@@ -25,7 +25,7 @@
 [submodule "mongodb_exporter"]
 	path = sources/mongodb_exporter/src/github.com/percona/mongodb_exporter
 	url = https://github.com/percona/mongodb_exporter.git
-	branch = 0.11.0
+	branch = PMM-6377
```
Comment (BupycHuk) [inline .gitmodules:28]: already merged.

**Review by BupycHuk — APPROVED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1058-d310358
client docker - perconalab/pmm-client-fb:PR-1058-d310358
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1058-d310358.tar.gz

**Comment by AlekSi** [comment]
@BupycHuk On you. I propose to merge it ASAP to unblock future work – even if that will break some MongoDB functionality until pmm-managed PR is merged, etc

/cc @alexanderblinkov-percona

**Review by AlekSi — APPROVED** [review]

**Review by EvgeniyPatlan — APPROVED** [review]

**Comment by BupycHuk** [comment]
Let's wait until jenkins job finished to see what will be broken.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1058-bd8473d
client docker - perconalab/pmm-client-fb:PR-1058-bd8473d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1058-bd8473d.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1058-fb3ea0a
client docker - perconalab/pmm-client-fb:PR-1058-fb3ea0a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1058-fb3ea0a.tar.gz

**Pull request merged**